reference/local.go

```diff
@@ -174,7 +174,7 @@ func (p *byteWriter) isFull() bool {
 
 // String implements stringer on ID and returns base58 encoded value
 func (v Local) String() string {
-	repr, err := DefaultEncoder().EncodeRecord(&v)
```

Comment:
everytime string method is called new base64 encoder is created? bad idea
BTW, all base64 record identifiers should be prefixed with schema (`EncodingSchema` flag or `FormatSchema`)